frontend/modelarchive/src/main/java/org/pytorch/serve/archive/ModelArchive.java

```diff
@@ -27,7 +28,7 @@
 
     public static final Gson GSON = new GsonBuilder().setPrettyPrinting().create();
 
-    private static final Pattern URL_PATTERN =
+    private static final Pattern VALID_URL_PATTERN =
             Pattern.compile("http(s)?://.*", Pattern.CASE_INSENSITIVE);
```

Comment:
@shivamshriwas In case of s3 urls, is it expected they will be in https://s3.xxxx format and not "s3://xxx"?

Reply:
Yes, 
This pattern is just used to verify whether given URL is a valid http / https  URL.